### ISSUE TITLE
Code formatting

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#449]
Some courses include code snippets. 

These should be rendered with appropriate HTML tags, and the styling should use a monospaced font. 

legacy example: https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-006-introduction-to-algorithms-fall-2011/syllabus/software/

nextgen example: https://ocwnext.odl.mit.edu/courses/6-006-introduction-to-algorithms-fall-2011/pages/syllabus/software/#ide

**Comment by pdpinch** [comment]
Looking at the markdown, I don't think we have enough data to identify the code snippets. This was probably lost in the import, so this may require work in ocw-to-hugo and ocw-hugo-themes.

**Comment by MAbdurrehman12** [comment]
>  so this may require work in ocw-to-hugo and ocw-hugo-themes.

Yeah, this will require work in `ocw-to-hugo` and in order to run a course and generate/download its markdown via `ocw-to-hugo` we will be either needing: AWS `access` key and `secret` or JSON of that course, but I think a read-only key will be much better

**Comment by MAbdurrehman12** [comment]
Got the key and secret from DevOps, I'll now look into this

**Comment by pdpinch** [comment]
Where are you seeing the inline styling? 

Instead of inline styling it would be better to convert them to the correct markdown, which I think is a [single backtick for inline code](https://www.markdownguide.org/basic-syntax#code) and [three backticks for blocks of code](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks).
